test/mpi/distributed/matrix.cpp

```diff
@@ -79,12 +79,13 @@ class MatrixCreation : public ::testing::Test {
     using Partition =
         gko::distributed::Partition<local_index_type, global_index_type>;
     using matrix_data = gko::matrix_data<value_type, global_index_type>;
+    using local_matrix_type = gko::matrix::Csr<value_type, local_index_type>;
 
 
     MatrixCreation()
         : ref(gko::ReferenceExecutor::create()),
           size{5, 5},
-          comm(gko::mpi::communicator(MPI_COMM_WORLD)),
+          comm(gko::mpi::communicator(MPI_COMM_WORLD, exec)),
```

Comment:
Again for these, wouldn't the following work?
```suggestion
          comm{MPI_COMM_WORLD, exec},
```